AdvancedDLSupport/Loaders/PlatformLoaderBase.cs

```diff
@@ -29,9 +29,10 @@ namespace AdvancedDLSupport.Loaders
     [PublicAPI]
     public abstract class PlatformLoaderBase : IPlatformLoader
     {
-        /// <inheritdoc />
-        public T LoadFunction<T>(IntPtr library, string symbolName) =>
-            Marshal.GetDelegateForFunctionPointer<T>(LoadSymbol(library, symbolName));
+        /// <summary>
+        /// Gets cached instance of the default Plaform Loader.

```

Comment:
Minor cleanup: write "Gets a cached instance of the current platform's default loader." instead.